string file_name
---
string cmd
bool result
